### DISCUSSION TITLE
I can't understand how to install yarn Berry

### DISCUSSION THREAD

**Discussion by brantac** [yarnpkg/berry#4519] [General]
I read the official yarn website, telling how to install, but I was not able to make it work.

To begin with, I wanted to make a fresh installation of yarn, so I uninstalled the version 1.x by doing ```rm -rfv ~/.yarn```, because I installed it using the script that downloads that tarball. And then I followed the instructions for installing the Berry version, but nothing happens.

My Node version is 16.3.1. I ran ```corepack enable```, and then I went to a project and ran ```yarn init -2```, and the result is ```bash: /home/<username>/.yarn/bin/yarn: No such file or directory```, because I think yarn is trying to run yarn using $PATH, but I deleted the files.

I think I'm not understanding how the "newest" yarn version works. Could someone try to explain for a newbie?

**Comment by milesj** [comment]
@brantac What does `which yarn` show? It looks like it's trying to find yarn in `.yarn`, where corepack installs it into the Node directory.

Is something adding the `.yarn` path to `PATH`? If you've installed yarn through something like homebrew or yvm, you should maybe delete those.

**Reply by brantac** [reply]
Hi @milesj , when I had yarn version 1.x, I ran ```which yarn``` and the path was ```/home/<username>/.yarn/bin/yarn```. I installed that version using ```curl -o- -L https://yarnpkg.com/install.sh | bash```, and I think I did manually what the [docs](https://classic.yarnpkg.com/en/docs/install#alternatives-stable) said:

![image](https://user-images.githubusercontent.com/22104847/171940063-d6dd46f3-477b-42a7-8f0b-87cf0fd76ccd.png)


But as I said, I "uninstalled" yarn 1.x by removing the directory, because my idea was to make a clean install of yarn 2.x.

And I ran ```which yarn``` now, this is what was printed to the terminal: ```bash: /home/<username>/.yarn/bin/yarn: No such file or directory```

And my $PATH has some yarn related paths: ```/home/<username>/.yarn/bin:/home/arthur/.config/yarn/global/node_modules/.bin```.

**Comment by brantac** [comment]
I think that I shouldn't have removed the .yarn directory. I installed yarn again using the script shown in the docs. Now I understand that to be able to use a version newer than 1.x, I just need to run ```yarn set version stable```. And if I want to use a specific version, I just need to switch "stable" by "specific-version".
